deploy/etc/nginx/vhosts/dolores.conf

```diff
@@ -0,0 +1,61 @@
+##
+# Dolores vhost
+# /etc/nginx/vhosts/dolores.conf
+##
+
+# HTTP server
+server {
+  listen 80;
+  server_name  dolores.18f.gov;
+  return 301 https://$host$request_uri;
+}
+
+# HTTPS server (with SPDY enabled)
+server {
+  listen 443 ssl spdy;
+  server_name  dolores.18f.gov;
+  include ssl/star.18f.gov.conf;
+  include vhosts/auth-locations.conf;
+}
+
+server {
+  listen 127.0.0.1:8080;
+  server_name dolores.18f.gov;
+  port_in_redirect off;
+
+  location / {
+    resolver 8.8.8.8;
+    set $backend_upstream "https://dolores.18f.gov";
```

Comment:
This should be the Cloud Foundry URL where the app will actually be deployed. (As it stands, it's going to try to proxy to itself.)